test/controllers/AgentTelephoneNumberControllerSpec.scala

```diff
@@ -16,7 +16,7 @@
 
 package controllers
 
-import base.SpecBase
+import base.{RegistrationSpecBase, SpecBase}
```

Comment:
Is SpecBase still used in these specs now that you're using RegistrationSpecBase?

Reply:
Updated